spec/test_app/db/migrate/20211013183057_add_missing_unique_indices.acts_as_taggable_on_engine.rb

```diff
@@ -0,0 +1,26 @@
+# This migration comes from acts_as_taggable_on_engine (originally 2)
```

Comment:
Do we need these migrations here as well as in the engine? The copy in the engine should get run in the host app.

Reply:
Removed 👍🏻